### ISSUE TITLE
Some CellDensityProfiles have invalid last boundary position

### ISSUE THREAD

**Issue by AhmetNSimsek** [FZJ-INM1-BDA/siibra-python#509]
If asked to get the data or plot, some CellDensityProfiles fail due to the last boundary position being just above 1.0 (eg  1.0000067544767959). This should be addressed either at the data level or siibra should handle these.

Some example ids:
```
dae2408a-95d0-4f63-ab37-f0756dff4720--07ee22d5b0824effaf90b0affc3d31aa
dae2408a-95d0-4f63-ab37-f0756dff4720--3411389421e087c6d398686a9e3c51d1
dae2408a-95d0-4f63-ab37-f0756dff4720--333a99f54a8116e04fa12d4b53fa9e7c
dae2408a-95d0-4f63-ab37-f0756dff4720--fd709c4fa7f3f7967e04065daee15799
16a71c7c-8940-4fc8-b2a7-d667933c2f2a--eb556c3f525337e305e68f71885325f3
f25c6728-2b53-4304-aa36-6165bbca3971--ee445f7352a3699813280ffd92207cb6
f25c6728-2b53-4304-aa36-6165bbca3971--0fbf3532f6d62f6f99f567a3c1102594
f25c6728-2b53-4304-aa36-6165bbca3971--aa9a723c9e15c02ea3f7fd02e374b3c4
f25c6728-2b53-4304-aa36-6165bbca3971--079376d3461c173c1d19139f556f3f71
```
Test with:
```python
import siibra
f = siibra.siibra.features._get_instance_by_id("dae2408a-95d0-4f63-ab37-f0756dff4720--07ee22d5b0824effaf90b0affc3d31aa")
f._check_sanity()
```

**Comment by xgui3783** [comment]
ditto for `lq0::BigBrainIntensityProfile::p:minds/core/parcellationatlas/v1.0.0/94c1125b-b87e-45e4-901c-00daee7f2579-300::r:Area hOc1 (V1, 17, CalcS) left::4c05163cac01b560cddf9d0ae2b63c94`

**Comment by AhmetNSimsek** [comment]
solved with #617
